### ISSUE TITLE
Match MAPS files to ToTS tapes and .TAPE files

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#1922]
In 1988, files were transferred from old 7-track to newer 9-track tapes and the MAPS, MAPS2, and MAPS3 directories recorded the files.  Many of those tapes are also in the ToTS collection, but the tape ID has no relation to the MAPS ID.  There are also .TAPE backup records which use the same ID as MAPS.

**Comment by larsbrinkhoff** [comment]
I have written a script to extract the tape header from the ToTS tapes.  This includes tape number and date, but not which machine the tape was created on.  I have also written another script to extract the same information from the "its-snapshot" MAPS files.  This does include machine.

I have matched the two datasets to add machine metadata for the ToTS tapes.  From this I can examine the .TAPE directories on the tapes for which I know the machine.

**Comment by larsbrinkhoff** [comment]
I have now written other scripts to compare the grand list of tapes with the .TAPE files on the tapes which in the previous round was matched to a single machine.  Using this iteratively to gradually expand the set of known tapes, I have been able to get around 50% coverage.  That is, for around half the tapes I have figured out which machine wrote it.

Most of the unknown tapes are from the 80s, which may be because I haven't extracted many of those yet.
